drivers/input/linux_evdev.c

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT linux_evdev
+
+#include <cmdline.h>
+#include <nsi_host_trampolines.h>
+#include <posix_native_task.h>
+#include <zephyr/device.h>
+#include <zephyr/input/input.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/util.h>
+
+#include "linux_evdev_bottom.h"
+
+LOG_MODULE_REGISTER(linux_evdev, CONFIG_INPUT_LOG_LEVEL);
+
+#define LINUX_EVDEV_READ_SLEEP_MS 10
```

Comment:
Maybe make this one a kconfig option?

Reply:
I really can't imagine why one would bother changing it, but since both you and @aescolar pointed it out, I made it I a config :-)